tools/sklearn_tools/discriminant.xml

```diff
@@ -0,0 +1,199 @@
+            <param name="solver" value="svd" />
+            <param name="store_covariances" value="True" />
+            <output name="outfile" file="linear_model01.txt"/>
+        </test>
+        <test>
+            <param name="infile_train" value="train_set.tabular" ftype="tabular"/>
+            <param name="selected_task" value="train"/>
+            <param name="selected_algorithm" value="LinearDiscriminantAnalysis"/>
+            <param name="solver" value="lsqr"/>
+            <output name="outfile" file="linear_model02.txt"/>
+        </test>
+        <!--test>
+            <param name="infile_train" value="train_set.tabular" ftype="tabular"/>
+            <param name="selected_task" value="train"/>
+            <param name="selected_algorithm" value="LinearDiscriminantAnalysis"/>
+            <param name="solver" value="eigen" />
+            <output name="outfile" file="linear_model03.txt"/>
+        </test-->
+        <test>
+            <param name="infile_train" value="train_set.tabular" ftype="tabular"/>
+            <param name="selected_task" value="train"/>
+            <param name="selected_algorithm" value="QuadraticAnalysis"/>
+            <output name="outfile" file="quadratic_model01.txt"/>
+        </test>
+        <test>
+            <param name="infile_model" value="linear_model01.txt" ftype="txt"/>
+            <param name="infile_data" value="test_set.tabular" ftype="tabular"/>
+            <param name="selected_task" value="load"/>
+            <output name="outfile" file="linear_prediction_result01.tabular"/>
+        </test>
+        <test>
+            <param name="infile_model" value="linear_model02.txt" ftype="txt"/>
+            <param name="infile_data" value="test_set.tabular" ftype="tabular"/>
+            <param name="selected_task" value="load"/>
+            <output name="outfile" file="linear_prediction_result02.tabular"/>
+        </test>
+        <!--test>
+            <param name="infile_model" value="linear_model03.txt" ftype="txt"/>
+            <param name="infile_data" value="test_set.tabular" ftype="tabular"/>
+            <param name="selected_task" value="load"/>
+            <output name="outfile" file="linear_prediction_result03.tabular"/>
+        </test-->
+        <test>
+            <param name="infile_model" value="quadratic_model01.txt" ftype="txt"/>
+            <param name="infile_data" value="test_set.tabular" ftype="tabular"/>
+            <param name="selected_task" value="load"/>
+            <output name="outfile" file="quad_prediction_result01.tabular"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+**What it does**
+Linear Discriminant Analysis and Quadratic Discriminant Analysis are two classic classifiers, with, as their names suggest, a linear and a quadratic decision surface, respectively. This module uses sklearn.discriminant_analysis package to implement Discriminant Analysis classification.
```

Comment:
A little bit information about the input and outputs would be nice. Format, scipy, can be used by ...